### PR TITLE
redo match-engagements-submissions-auth.feature

### PR DESCRIPTION
**Pull request by alvaradom0321**
## Describe context / purpose for this PR
Added TODO testing

## Issue link
https://cyberteamportal.atlassian.net/browse/D8-1572

## Link to MultiDev instance

http://md-{{ ticket number }}-accessmatch.pantheonsite.io

## Checklist for PR author
- [ ] I have checked that the PR is ready to be merged
- [x] I have reviewed the DIFF and checked that the changes are as expected
- [x] I have assigned myself or someone else to review the PR

### CODE REVIEW

File: tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature [hunk 1]
```diff
@@ -2,30 +2,18 @@
...
-  The MATCH Engagement Submissions page has a table listing submissions
-  This page should be viewable by match_pm and match_sc roles TODO: but not authenticated
-  Displaying:
-  - Link to edit
-  - Status
-  - Title with link to view
-  - Team with links to community persona for researcher(s), student(s), mentor(s), and consultant
-  - Launch presentation date
-  - Wrap presentation date
+Feature: testing match engagements submissions for Match SC role
```
Comment (jasperlieber) [inline tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature:6]: hi @alvaradom0321  -- we don't want to lose all these lines of description of what the test should do.

File: tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature [hunk 2]
```diff
@@ -42,11 +30,17 @@ Feature: MATCH Engagements submissions page
...
+    When I am on "/match-engagements-submissions"
+    Then I should see "Team"
+    Then I should see "Researcher: Xiaoqin Huang"
+    Then I should see "Mentor: Xiaoqin Huang"
+    Then I should see "Steering Committee: Alana Romanella"
+    Then I should see "Students: Alexandra Lamtyugina"
+
 
-  Scenario: MATCH PM role test match engagements submissions page
+  Scenario: Admin with Match PM role test match engagements submissions page
```
Comment (jasperlieber) [inline tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature:43]: This should probably be written as "User with ..."  not "Admin with ..." .  The test is not running as an administrator, so this is a bit misleading.

File: tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature [hunk 3]
```diff
@@ -42,11 +30,17 @@ Feature: MATCH Engagements submissions page
...
-    #TODO: test team column displays mentor, student, consultant, and steering committee assignments
+    #TODO: test team column displays consultant
     # see https://cyberteamportal.atlassian.net/browse/D8-1593
+    When I am on "/match-engagements-submissions"
+    Then I should see "Team"
+    Then I should see "Researcher: Xiaoqin Huang"
+    Then I should see "Mentor: Xiaoqin Huang"
+    Then I should see "Steering Committee: Alana Romanella"
+    Then I should see "Students: Alexandra Lamtyugina"
+
```
Comment (jasperlieber) [inline tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature:41]: @a-pasquale -- do we want to test for a specific engagement like this?  my hunch is we should create one, and verify it looks good, yes?

**Review by jasperlieber — CHANGES_REQUESTED** [review]

File: tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature (hunk 3 above)
Comment (a-pasquale) [reply tests/behat/features/asp/engagements/match-engagements-submissions-auth.feature:41]: This is fine for the short term. We can revisit and create an engagement with behat or amp_dev in the future.

**Comment by jasperlieber** [comment]
redone on https://github.com/necyberteam/cyberteam_drupal/pull/849

**Pull request closed without merging**